Fix grammar error

garnerrobert opened the pull request:
Proposing a fix to a grammatical error on the Members and Donors page in English ('How become a sustaining member?' --> 'How to become a sustaining member?')

mbernasocchi commented:
as per PSC vote, **Temporarily** remove the links to iOS app store until a decision is reached on a redistribution amendment to the QGIS license
https://docs.google.com/document/d/1jINPb0dQuW2Jqazuux2a_87PjEaO8OSgc8zGb5zCGGE

garnerrobert committed:
Fix grammar error

Proposing a fix to a grammatical error (How become a sustaining member? --> How to become a sustaining member?)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/qgis/QGIS-Website?pullRequest=735) <br/>All committers have signed the CLA.

Xpirix reviewed (approved):
LGTM, thanks @garnerrobert .
